src/source_controller/coreservice/core/hostapplyrule/plan.go

```diff
@@ -69,8 +70,10 @@ func (p *hostApplyRule) GenerateApplyPlan(kit *rest.Kit, bizID int64, option met
 	}
 
 	hosts := make([]metadata.HostMapStr, 0)
-	if err := mongodb.Client().Table(common.BKTableNameBaseHost).Find(hostFilter).Fields(fields...).All(kit.Ctx, &hosts); err != nil {
-		blog.ErrorJSON("GenerateApplyPlan failed, list hosts failed, filter: %s, err: %s, rid: %s", hostFilter, err.Error(), rid)
+	if err := mongodb.Client().Table(common.BKTableNameBaseHost).Find(hostFilter).Fields(fields...).All(kit.Ctx,
+		&hosts); err != nil {
+		blog.ErrorJSON("GenerateApplyPlan failed, list hosts failed, filter: %s, err: %s, rid: %s", hostFilter,
+			err.Error(), rid)
```

Comment:
日志要按规范改，不能只是换个行，尽量不用ErrorJSON，然后只描述哪个操作报错了，不要带多余的函数名。下面那几个也都按规范改
```suggestion
		blog.Errorf("list hosts failed, filter: %+v, err: %v, rid: %s", hostFilter, err, rid)
```